Lib/test/test_asyncio/test_unix_events.py

```diff
@@ -1824,8 +1824,6 @@ def test_get_child_watcher_after_set(self):
     def test_get_child_watcher_thread(self):
 
         def f():
-            policy.set_event_loop(policy.new_event_loop())
```

Comment:
What about:

```
def f():
    policy.set_event_loop(policy.new_event_loop())
    policy.set_child_watcher(asyncio.SafeChildWatcher())
```

Reply:
Changed to keep `policy.set_event_loop()` in a thread but setup the watcher in the main thread (it is crucial for the test).